### PR TITLE
Update TUIAttachmentUpload

### PR DESCRIPTION
**Pull request by maheshwaran01m**

### CODE REVIEW

File: Sources/TarkaUI/Components/Attachment/TUIAttachmentUpload.swift [hunk 1]
```diff
@@ -113,6 +119,42 @@ public struct TUIAttachmentUpload: View {
 
 public extension TUIAttachmentUpload {
   
+  struct InputItem {
```
Comment (arvindhsukumar) [inline Sources/TarkaUI/Components/Attachment/TUIAttachmentUpload.swift:122]: I feel the model and it's styling attributes such as size and color should be in separate objects

Comment (maheshwaran01m) [reply Sources/TarkaUI/Components/Attachment/TUIAttachmentUpload.swift:122]: @arvindhsukumar if we have more objects, it makes some confusions, shall we name some thing else?

Comment (arvindhsukumar) [reply Sources/TarkaUI/Components/Attachment/TUIAttachmentUpload.swift:122]: Sure.. Anything other than InputItem, which to me indicates a model object, not styling properties

Comment (maheshwaran01m) [reply Sources/TarkaUI/Components/Attachment/TUIAttachmentUpload.swift]: @arvindhsukumar Updated

**Review by arvindhsukumar — APPROVED** [review]

**Review by gopinathcholan — APPROVED** [review]

**Pull request merged**